src/Scene.cpp

```
@@ -2,10 +2,12 @@
 #include <fstream>
 #include <iostream>
 #include <sstream>
+#include "ParserConfigFile.hpp"
 #include "Ray.hpp"
 #include "Renderer.hpp"
 
-Raytracer::Scene::Scene(int width, int height) {
+Raytracer::Scene::Scene(int width, int height, const std::string &inputPath) {
+  _path = inputPath;
```

Cppcheck (2.10-2)

[performance] Variable '_path' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)